<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>喵喵隐者4</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/ns4.png</normaloff>:/images/ns4.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="copyLeft">
        <property name="text">
         <string>复制原文</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pasteLeft">
        <property name="text">
         <string>粘贴原文</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearLeft">
        <property name="text">
         <string>清空原文</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="left">
      <property name="placeholderText">
       <string>原文</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="copyRight">
        <property name="text">
         <string>复制密文</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pasteRight">
        <property name="text">
         <string>粘贴密文</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearRight">
        <property name="text">
         <string>清空密文</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="right">
      <property name="placeholderText">
       <string>密文</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
